docs/source/examples/supplychain/overview.rst

```diff
@@ -0,0 +1,100 @@
+*******************
+SupplyChain Example
+*******************
+
+In this example provided the infrastructure necessary track physical objects
+thru a real world supply chain on the Sawtooth Ledger. Obviously we can not put
```

Comment:
Question: why is there an **Overview** document and a main document?
Also, regarding the use of "their" or "they" as a gender-neutral singular pronoun, it turns out I was wrong. It recently became accepted to use "they/their" as a gender-neutral, singular pronoun. Such is the evolution of language. ;)